Classifier.py

```diff
@@ -4,80 +4,40 @@
 """
 Author: Daryna Ivanova.
 
-Purpose: data preprocessing.
-
+Purpose: Implementation of syntactic features: number of function words.
```

Comment:
Wichtige Informationen, die noch im Header stehen könnten, wären zB deine verwendete Python Version und das Betriebssystem auf dem der Code getestet ist. Grund dafür ist, dass sich Betriebssysteme in einigen Dingen unterscheiden, die dafür sorgen können, dass ein Programm vielleicht auf Windows 10 funktioniert aber nicht auf Ubuntu.